web/src/stream_edit.ts

```diff
@@ -730,7 +821,7 @@ export function initialize() {
             }
             dialog_widget.launch({
                 html_heading: $t_html({defaultMessage: "Confirm changing access permissions"}),
-                html_body: render_confirm_stream_privacy_change_modal,
+                html_body: render_confirm_stream_privacy_change_modal(),
```

Comment:
prep to change this, with a bit more investigation into why it was working before